File: C_Programming/Even_odd.cpp
#include<stdio.h>
int main()
{
	int num;
	printf("Enter Any Number\n");
	scanf("%d",&num);
	switch (num%2)
	{
		case 0:
		printf("Number is Even");
		break;
		
		case 1:
		printf("Number is Odd");
		break;
		}
}

File: C#/Assignments/even_odd.cs
using System;
namespace EVENODD
{
    class Evenodd
    {
        public static void Main(string[] args)
        {
            int num;
            Console.Write("Enter Number: ");
            num = Convert.ToInt32(Console.ReadLine());

            if(num % 2 == 0) 
            {
                Console.WriteLine("Number are Even");
            }
            else
            {
                Console.WriteLine("Number are Odd");
            }
            Console.ReadKey();
        }
    }
}